### PR TITLE
15. As a user, I want a clean item card relevant data for each item on my list based on how it's sorted, and to be able to "purchase" items and remove them

### LINKED ISSUE #34: 15. As a user, I want a clean item card relevant data for each item on my list based on how it's sorted, and to be able to "purchase" items and remove them

### PR DESCRIPTION
**Pull request by thekidnamedkd**
## Description

This PR will style each item card to match the wireframe layout and design choices.

## Related Issue

Closes #34 

## Acceptance Criteria

* should display days to purchase
* should display last purchase date
* should have a button to checkoff or "purchase" the item, this will handle updated data and stay checked for 24 hours
* should have a button to remove the item
* should color code according to the window that days to purchase falls in

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
| ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before
Old Item List

### After
New Item List

### CODE REVIEW

**Review by Maeesha-Rahman — APPROVED** [review]
This is looking really good! Matches the mockup well! ✨

File: src/components/ShoppingList/PopulatedList/PopulatedList.js [hunk 1]
```diff
@@ -206,11 +207,8 @@ const PopulatedList = () => {
                           checked={hasItemBeenPurchased(
                             groceryItem.lastPurchasedDate,
                           )}
-                          visible="false"
                         />
-                        <label for="checkbox">
-                          <Checkbox className="circle" aria-hidden="true" />
-                        </label>
```
Comment (Maeesha-Rahman) [inline src/components/ShoppingList/PopulatedList/PopulatedList.js]: I think `for` should be `htmlFor` here

File: src/components/ShoppingList/PopulatedList/PopulatedList.js [hunk 2]
```diff
@@ -206,11 +207,8 @@ const PopulatedList = () => {
...
+                        <label for={groceryItem.itemName} />
```
Comment (Maeesha-Rahman) [inline src/components/ShoppingList/PopulatedList/PopulatedList.js]: I think `for` should be `htmlFor` here

**Review by jessicasalbert — APPROVED** [review]
Looks great!!! Thanks for tackling this big one.

File: src/components/ShoppingList/PopulatedList/PopulatedList.js [hunk 3]
```diff
@@ -143,58 +145,103 @@ const PopulatedList = () => {
...
+      9: "Oct.",
+      10: "Nov.",
+      11: "Dec.",
+    };
+    const timestamp = new Date(seconds * 1000);
+    return `${
+      months[timestamp.getMonth()]
+    } ${timestamp.getDate()}, ${timestamp.getFullYear()}`;
+  }
+
```
Comment (wlcreate) [inline src/components/ShoppingList/PopulatedList/PopulatedList.js:168]: Awesome job with this function to format our date!

**Review by wlcreate — APPROVED** [review]
This was a big issue and you did an amazing job!

File: src/components/ShoppingList/PopulatedList/PopulatedList.js (hunk 3 above)
Comment (Maeesha-Rahman) [reply src/components/ShoppingList/PopulatedList/PopulatedList.js:168]: Agreed!! Great work @thekidnamedkd 😄

**Pull request merged**